specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json

```diff
@@ -2071,6 +2071,10 @@
           "$ref": "#/definitions/RedisCreateProperties",
           "description": "Redis cache properties."
         },
+        "redisVersion": {
+          "type": "string",
+          "description": "Redis version."
```

Comment:
We should have information about which version values are 'allowed' in the Description, in order to help people using the known good version number which matches their expected semantics. Currently '4.0' and '6.0' right?

Reply:
Yes, currently we support 4 and 6. Updated description and added enum to enforce the validaiton.